release/pie-chart/pie-grid.component.d.ts

```diff
@@ -3,6 +3,7 @@ import { ViewDimensions } from '../common/view-dimensions.helper';
 import { ColorHelper } from '../common/color.helper';
 import { BaseChartComponent } from '../common/base-chart.component';
 export declare class PieGridComponent extends BaseChartComponent {
+    designatedTotal: number;
```

Comment:
Let's remove the change from the release files please. They are autogenerated during release, and should not be committed to the pull request.